rustls/src/msgs/macros.rs

```diff
@@ -59,6 +59,13 @@ macro_rules! enum_builder {
                     ,$enum_name::Unknown(x) => x
                 }
             }
+
+            pub fn as_str(&self) -> Option<&'static str> {
```

Comment:
By adding this as another method generated by the `enum_builder!()` implementation for `U16` only we get this for 6 of the defined enums (including `CipherSuite`). This seems preferable to writing the code separately as an inherent impl for `CipherSuite` (since it would require repeating all the variants -> lots of boilerplate). Could additionally do this for `U8` enums, but it feels like we would just be generating more code that is probably not needed.

Chose to return `Option<u8>` so we can return `None` for the `Unknown` case, which seems most useful. Otherwise we could return `unknown` or change the return type to be `Cow<'static, str>` and format into `Unknown(13)` instead.